docs/source/trainer/checkpointing.rst

```diff
@@ -379,11 +379,56 @@ There are a few additional trainer arguments which can be helpful to configure:
     will remove each checkpoint after it has been uploaded. Checkpoints are never deleted from object stores.
 
 
-Loading from Object Store
--------------------------
+Saving Checkpoints to Google Cloud Storage (GCS)
+------------------------------------------------
+
+To save checkpoints to GCS, make sure to create a :class:`.RemoteUploaderDownloader` instance
+and pass it to the `Trainer` loggers argument.
+
+.. testcode::
+    :skipif: not _LIBCLOUD_INSTALLED
+
+    from composer.loggers import RemoteUploaderDownloader
+    from composer.trainer import Trainer
+
+    remote_uploader_downloader = RemoteUploaderDownloader(
+        bucket_uri="libcloud://my-gcs-bucket",
+        backend_kwargs={
+            "provider": "google_storage",  # The Apache Libcloud provider name
+            "container": "my-gcs-bucket",  # The name of the cloud container (i.e. bucket) to use.
```

Comment:
This arg shouldn't actually have to be provided, as it will be parsed from `bucket_uri`, but no harm leaving it in the example i think?

Reply:
yeah?